### DISCUSSION TITLE
A default world button

### DISCUSSION THREAD

**Discussion by thatonelilguyfromtechtok** [runelite/runelite#18526] [Ideas]
Having a default world button like on the official client would be nice because every time I launch the game its selecting a world randomly so this would be nice.
Thanks
Noah

**Comment by Bram91** [comment]
You can achieve this with the Default World plugin: https://github.com/runelite/runelite/wiki/Default-World

**Reply by thatonelilguyfromtechtok** [reply]
thanks

**Reply by thatonelilguyfromtechtok** [reply]
idk if its normal but there is nothing on the page
![image](https://github.com/user-attachments/assets/ba82e3d7-ed4c-4f06-af0a-3825ce1421c5)

**Reply by cdfisher** [reply]
Open it in the sidebar in the client

**Reply by thatonelilguyfromtechtok** [reply]
thanks
